src/views/evaluation/Evaluation.js

```diff
@@ -112,11 +102,26 @@ window.onload = function(){
                     tHeader : ["Nume", "Tehnologie", "Nivel"],
                     ImageLink: "/assets/button.jpg",
                     }
-    footerContent = {Footer : "SoftVision@2017"}
-    headerContent = {Logo : "/assets/logo-v2.jpg",
-                    Link: ["Evaluationv2.html", "NewEvaluationv2.html", "Loginv2.html"],
-                    Text: ["Evaluation", "New Evaluation", "Logout"]}
-    option = {tContent : tableContent, fContent : footerContent, hContent : headerContent}
+    __footerContent = {Footer : "SoftVision@2017"}
+
+    __headerContent = [{
+        Link: "Evaluationv2.html",
+        Text: "Evaluation"
+    },
+    {
+        Link: "NewEvaluationv2.html",
+        Text: "New Evaluation"
+    },
+    {
+        Link: "Loginv2.html",
+        Text: "Logout"
+    }]
+
+
+    __logoContent = {Link : "assets/logo-v2.jpg",
+    Alt: "Company logo"}
+
+    option = {tContent : tableContent, footerContent : __footerContent, logoContent : __logoContent, headerContent : __headerContent}
```

Comment:
I would concentrate on getting the bulk of the implementation done first, at this point in time. Whether it makes more sense to use a `.map` vs a `.forEach` or a standard `for` / `while` loop instead is usually part of code optimization and I wouldn't want to have you guys thinking too much about optimizations at this point in time. We want you to get comfortable with the basics during this training, then we'll start piling other more advanced concepts on top of that once you're done absorbing said basics.